Disable "main_context_futures" tests on arm32, they cause the testsuite
to fail erratically with "signal: 6, SIGABRT: process abort signal",
there is no obvious link between cause and effect, the problem was
narrowed down by painful trial and error.

Initially these tests were only disabled on armel, but I have since seen similar
sporadic testsuite crashes on armhf, so I am skipping them on all arm32 systems

Index: rust-glib-0.17.10/src/main_context_futures.rs
===================================================================
--- rust-glib-0.17.10.orig/src/main_context_futures.rs
+++ rust-glib-0.17.10/src/main_context_futures.rs
@@ -687,6 +687,7 @@ impl LocalSpawn for MainContext {
 }
 
 #[cfg(test)]
+#[cfg(not(target_arch = "arm"))]
 mod tests {
     use std::{sync::mpsc, thread};
 
